Makefile

```diff
@@ -44,7 +44,7 @@ lint:  ## Run a `lint` process on Hera and report problems
 
 .PHONY: test
 test:  ## Run tests for Hera
-	@poetry run python -m pytest --cov-report=term-missing
+	@poetry run python -m pytest --cov-report=term-missing -m "not workflow"
```

Comment:
picky comment - would this flag be more intuitive if it were called "remote" or "remote_in_ci"? A bunch of tests test workflows

Reply:
Yeah lol I didn't want to get hung up on the naming, STC! 😆 Will change